src/style.css

```diff
@@ -1 +1,24 @@
-/* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Don't use `*` selector as it impacts performance. It's better to set styles specifically for elements that require them. If you need to reset margins and paddings, consider using a more specific reset rule or a reset file.